.travis.yml

```diff
@@ -4,44 +4,72 @@ node_js:
   - "lts/*"
   - "stable"
 
+git:
+  clone: false # If you use this option, the TRAVIS_COMMIT_MESSAGE environment variable will not be defined.
```

Comment:
The comments In romcal codebase usually does not end with a dot (full stop, `.`).

Reply:
Done!